docs/advanced/prometheus.md

```diff
@@ -0,0 +1,227 @@
+---
+title: Monitoring KubeEdge Edge Nodes with Prometheus
+sidebar_position: 6
+---
+# Monitoring KubeEdge Edge Nodes with Prometheus
+
+## Environment Information
+
+| Component  | Version                            |
+| ---------- | ---------------------------------- |
+| containerd | 1.7.2                              |
+| k8s        | 1.26.0                             |
+| KubeEdge   | 1.15.1 or 1.17.0                   |
```

Comment:
This may be misleading to users, as it may imply that only these two versions are supported. Would it be possible to consider adding an explanation here?  like: this feature is recommended for version 1.15.0 and above. Since v1.17.0 supports edge pods using `InclusterConfig`, the approach is different for versions before and after v1.17.0. This document will use v1.15.1 and v1.17.0 as examples to illustrate the steps.

Reply:
下面增加了 关于版本的说明